src/main/java/frc/robot/subsystems/swerve/module/Module.java

```diff
@@ -160,13 +160,17 @@ public void setSteerVoltage(double voltage) {
 
 	public void pointSteer(Rotation2d steerTargetPosition, boolean optimize) {
 		SwerveModuleState moduleState = new SwerveModuleState(0, steerTargetPosition);
-		targetState.angle = optimize ? targetState.optimize(moduleState, getSteerPosition()).angle : moduleState.angle;
+		if (optimize) {
+			targetState.optimize(getSteerPosition());
```

Comment:
don't you mean moduleState.optimize?

Reply:
No such variable, this is the module state